src/bindings/python/flux/Makefile.am

```diff
@@ -40,6 +40,8 @@ nobase_fluxpy_PYTHON = \
 	hostlist.py \
```

Comment:
commit message s/hiearchical/hierarchical/

> if a URI is resolved with no scheme, it is assumed to be a Flux jobid

Did you mean "if a URI is parsed with no scheme..."?  And is the default scheme an optimization or a convenience?

Reply:
> And is the default scheme an optimization or a convenience?

Yeah, a convenience. I'll just drop the `As an optimization,` -- no need for filler phrases in the commit message